libvast/src/system/version_command.cpp

```diff
@@ -53,6 +57,11 @@ json::object retrieve_versions() {
   result["PCAP"] = pcap_lib_version();
 #else
   result["PCAP"] = json{};
+#endif
+#if VAST_USE_JEMALLOC
+  result["JEMALLOC"] = JEMALLOC_VERSION;
+#else
+  result["JEMALLOC"] = json{};
```

Comment:
I know this is all uppercase because all the other things are, but do you mind changing `ARROW` to `Apache Arrow` and `JEMALLOC` to `jemalloc`?